bin/strap.sh

```diff
@@ -298,6 +300,17 @@ if git ls-remote "$DOTFILES_URL" &>/dev/null; then
   fi
   (
     cd "$DOTFILES_DIR"
+    CURRENT_DAPTIV_DOTFILES_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+    if [ "$DAPTIV_DOTFILES_BRANCH" != "$CURRENT_DAPTIV_DOTFILES_BRANCH" ]; then
+      # check to make sure there are no pending changes in current branch
+      if git diff-index --quiet HEAD -- ; then
+        log "Changing branch from '$CURRENT_DAPTIV_DOTFILES_BRANCH' to '$DAPTIV_DOTFILES_BRANCH'"
+        git checkout $DAPTIV_DOTFILES_BRANCH
```

Comment:
are you sure you want to move folks off their branch?  it might be useful to test a branch before pushing

Reply:
it will only switch if the requested branch and current branch don't match.